Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml

```diff
@@ -709,54 +717,7 @@ tasks:
       {
         "position": {
           "x": 950,
-          "y": 1030
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
-    skipunavailable: false
-    quietmode: 0
-    isoversize: false
-    isautoswitchedtoquietmode: false
-  "21":
-    id: "21"
-    taskid: d747cea6-9883-4960-8c1a-5bb9f22fbeb7
-    type: regular
-    task:
-      id: d747cea6-9883-4960-8c1a-5bb9f22fbeb7
-      version: -1
-      name: Set system IDs grid field (firewall)
```

Comment:
Is there a reason not to store this anymore? Would it still not be useful information for the customer?

Reply:
So in https://github.com/demisto/content/pull/32678 i am actually replacing the !gcp-compute-list-firewall command with the GCPOffendingFirewallRule automation that uses that command.  Before we were storing all GCP firewall rules in a particular network in that system IDs gridfield which I don't think is useful.